lua/pac3/core/client/base_part.lua

```diff
@@ -272,7 +276,7 @@ do -- scene graph
 		-- this will handle cases like if a part is removed and added again
 		for _, key in pairs(self.PartReferenceKeys) do
 			if self[key] and self[key].UniqueID == part.UniqueID then
-				self["Set" .. key](self, part)
+				self[prefix_set .. key](self, part)
```

Comment:
Making this a variable is pointless. Getting rid of the concatenation would be a good performance improvement.

Reply:
Also I don't like making these constant strings into variables since it fragments and complicates the code.